### PR TITLE
Fix issues identified by automated audit

### LINKED ISSUE #17: Recommended changes resulting from automated audit
We performed an automated audit of your Cake addin and found that it does not follow all the best practices.

We encourage you to make the following modifications:

- [ ] You are currently referencing Cake.Core 0.28.0. Please upgrade to 0.33.0
- [ ] The nuget package for your addin should use the cake-contrib icon. Specifically, your addin's `.csproj` should have a line like this: `<PackageIconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics/png/cake-contrib-medium.png</PackageIconUrl>`.

Apologies if this is already being worked on, or if there are existing open issues, this issue was created based on what is currently published for this package on NuGet.

This issue was created by a tool: Cake.AddinDiscoverer version 3.12.1

### PR DESCRIPTION
**Pull request by Jericho**
This pull request was created by a tool: Cake.AddinDiscoverer version 3.14.0

Resolves #17

### CODE REVIEW

File: src/Cake.Markdownlint/Cake.Markdownlint.csproj [hunk 1]
```diff
@@ -23,8 +23,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="0.28.0">
-      <PrivateAssets>all</PrivateAssets>
+    <PackageReference Include="Cake.Core" Version="0.33.0">
+      <PrivateAssets>All</PrivateAssets>
```
Comment (Jericho) [inline src/Cake.Markdownlint/Cake.Markdownlint.csproj]: I just noticed that AddinDiscoverer changed the casing of the `PrivateAssets` value. I don't know if the value is case-sensitive or not. If so, what is the proper casing? 

This [example](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets) seems to indicate that it should be all lower-case but the article does not specifically address case-sensitivity.

Comment (Jericho) [reply src/Cake.Markdownlint/Cake.Markdownlint.csproj]: In this other [article](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packagereference), the first letter in `All` is capitalized.

Comment (Jericho) [reply src/Cake.Markdownlint/Cake.Markdownlint.csproj]: Investigating this situation on GitHub, I found a mix of "all" and "All": https://github.com/search?q=PrivateAssets+all&type=Code

This leads me to conclude that this value is not case sensitive and therefore your lower-case value should not have been changed by the AddinDiscoverer. I will fix this PR to restore your original value.

**Comment by pascalberger** [comment]
This was fixed with #19

**Pull request closed without merging**